Program.cs

```diff
@@ -2,60 +2,110 @@
 
 namespace objected_oriented_programming
 {
-   abstract class MOSGORGEOTREST
+    enum typesOfBankAccounts
```

Comment:
Модификатор доступа потеряли. 
```suggestion
    private enum typesOfBankAccounts
```